docs/MAVLINKX.md

```diff
@@ -0,0 +1,21 @@
+# mLRS Documentation: MavlinkX #
+
+([back to main page](../README.md))
+
+MavlinkX is a technology to improve upon the original MAVLink protocol. It introduces robust framing and parsing which reduces packet losses to the minimum, and adds per-message compression. Both of these features allow for more efficient use of the link capacity.
```

Comment:
Perhaps:

'MavlinkX is a technology designed to improve upon the original MAVLink protocol.'

Reply:
great. done.